src/BlazorUI/Bit.BlazorUI.Tests/SnackBar/BitSnackBarWithTitleTemplate.razor

```diff
@@ -0,0 +1,6 @@
+<BitSnackBar>
+    <TitleTemplate Context="title">
+        <span>@title</span>
+        <span>second element</span>
```

Comment:
again, either remove this line or accept it as a parameter, since it is part of the expected value in the test.